### PR TITLE
PMM-3714 API for QAN MySQL Slowlog agent.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-332/

File: api/inventorypb/agents.proto [hunk 1]
```diff
@@ -181,6 +181,30 @@ message QANMySQLPerfSchemaAgent {
...
+  // Status fields below.
+  //
+
+  // Desired Agent status: enabled (false) or disabled (true).
+  bool disabled = 6;
+  // Actual Agent status.
+  AgentStatus status = 7;
+  // Custom user-assigned labels.
+  map<string, string> custom_labels = 10;
+}
```
Comment (AlekSi) [inline api/inventorypb/agents.proto]: We should also add it to Get and List.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-332/

**Comment by BupycHuk** [comment]
@AlekSi do you mean enum? It's already there.

**Pull request merged**